tools/online_editor/package.json

```diff
@@ -5,20 +5,25 @@
   "main": "index.js",
   "scripts": {
     "build": "rimraf dist pkg && npx vite build",
-    "build:wasm_preview": "wasm-pack build --target web ../../api/wasm-interpreter",
-    "lint": "eslint *.ts",
-    "start": "rimraf dist && npm run build:wasm_preview && npn start:vite",
+    "build:wasm_preview-release": "wasm-pack build --release --target web ../../api/wasm-interpreter -- --features console_error_panic_hook",
+    "build:wasm_lsp-release": "wasm-pack build --release --target web ../lsp -- --no-default-features",
+    "build:wasm_preview": "wasm-pack build --target web ../../api/wasm-interpreter -- --features console_error_panic_hook",
+    "build:wasm_lsp": "wasm-pack build --target web ../lsp -- --no-default-features",
+    "lint": "eslint src",
+    "start": "rimraf dist && npm run build:wasm_preview && npm run start:vite",
```

Comment:
Looks like this also need `npm run build:wasm_lsp`

Reply:
You are right! Thanks!